ang/civicase/daypickerDecorator.js

```diff
@@ -0,0 +1,42 @@
+(function (angular) {
+  var module = angular.module('civicase');
+
+  module.config(function ($provide) {
+    $provide.decorator('uibDaypickerDirective', function ($controller, $delegate) {
+      var datepicker = $delegate[0];
+
+      datepicker.controller = function ($scope, $element, dateFilter) {
+        var vm = this;
```

Comment:
Can we use `$scope` here? instead of using `this`?

Reply:
`vm` is used to extend the controller function, not the `$scope`. This is needed because the uib day picker controller defines a few functions as part of the function and not the `$scope`.